README.md

```diff
@@ -24,6 +24,14 @@ A 500 response body will be returned if any issues occur when calling the Lambda
 * numberofrefs parameter isn't an int
 * any dynamodb Exceptions (key not found, unable to update, etc)
 
+The number of references that can be returned in a single call is limited, because:
+* prevent a single call using up all possible reference;
+* limits to the permitted size of the response.
+
+Calling clients will need to handle this limit by making multiple calls to retrieve the required number of references if it is greater than the limit.
+
+The limit is store here for use by calling clients: [reference_generator_limit](https://github.com/nationalarchives/da-terraform-configurations/blob/main/tdr/main.tf)
```

Comment:
A minor rewording here, change from `The limit is store here` to `The limit is stored here`